trusted-fingerprint/docker/app.py

```diff
@@ -32,9 +33,10 @@ def do_POST(self):
             return
 
         try:
-            transport = paramiko.Transport(host)
+            sock = socket.create_connection((host, 22), timeout=60)
```

Comment:
Can we close the socket connection once we are done? Maybe inside a `finally` block?

Reply:
We do close the connection, please refer line 44.

Reply:
We close the SSH Transport stream which ideally should close the socket as well. But since we are creating our own socket connection instead of providing the host and letting it manage the connections, it may not close the provided socket on its own. Better to be explicit and close it ourselves.

Reply:
Makes sense. Added